static/js/subs.js

```diff
@@ -431,6 +431,7 @@ exports.setup_page = function () {
 
     function populate_and_fill(public_streams) {
 
+        exports.all_streams = public_streams.streams;
```

Comment:
Why is `all_streams` added to `exports` here? It doesn't seem to be used outside of this module